utility/src/main/java/org/oppia/android/util/logging/EventBundleCreator.kt

```diff
@@ -129,6 +129,9 @@ class EventBundleCreator @Inject constructor(
   private val appVersionCode by lazy { context.getVersionCode() }
   private val appVersionName by lazy { context.getVersionName() }
   private val eventCount by lazy { AtomicInteger() }
+  private val screenDensity by lazy {
+    context.resources.displayMetrics.densityDpi
+  }
```

Comment:
From the docs for this API:
<img width="439" alt="Screenshot 2024-01-03 at 23 33 03" src="https://github.com/oppia/oppia-android/assets/59600948/54f57d4b-cebd-47da-8dd2-f5b8264370dc">

It looks like this value may be affected by the user's display settings, including display zoom.  The solution required should NOT be affected by display zoom, and according to the docs, we need to use `windowManager.defaultDisplay.getMetrics(metrics)`

 If you want to obtain the physical DPI without being affected by display zoom, you need to use the DisplayMetrics method xdpi (horizontal pixels per inch) and ydpi (vertical pixels per inch):
 
```Kotlin
val metrics = DisplayMetrics()
windowManager.defaultDisplay.getMetrics(metrics)
val xDpi: Float = metrics.xdpi
val yDpi: Float = metrics.ydpi

// Convert to integer as needed
val xDpiInt: Int = xDpi.roundToInt()
val yDpiInt: Int = yDpi.roundToInt()



```

Reply:
hey @adhiamboperes, do we need to log 2 values? Couldn't find any way to get the screen density in dpi using the xDpi and yDpi.

Reply:
@Tejas-67, I see your concern. I have referred again to the screenshot above and it is suggested that we can get density from  `Configuration().densityDpi`, and where the information is not available(this is possible), we can set  the value to `Configuration.DENSITY_DPI_UNDEFINED`

You can update this to: 
 private val screenDensity by lazy {
   Configuration().densityDpi
  }

Reply:
Please check section 2 and 3 of PR guidelines for future reference: https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR